src/js/cart.js

```diff
@@ -1,7 +1,7 @@
 import { getLocalStorage } from "./utils.mjs";
 
 function renderCartContents() {
-  const cartItems = getLocalStorage("so-cart");
+  const cartItems = getLocalStorage("so-cart") || [];
```

Comment:
Also from Week 1 bug fixes. Do not forget to incorporate them in your refactors.